target/personalization-client-side/README.md

```diff
@@ -109,4 +109,4 @@ sequenceDiagram
 
 ## Beyond the sample
 
-This sample app can serve as a starting point for you to experiment and learn more about Adobe Experience Platform. For example, you can change a few environment variables so the sample app pulls in offers from your own AEP configuration. To do so, just open the `.env` file at the root of this repository and modify the variables. Restart the sample app, and you're ready to experiment using your own personalization content.
+This sample app can serve as a starting point for you to experiment and learn more about Adobe Experience Platform. For example, you can change a few environment variables so the sample app pulls in offers from your own AEP configuration. To do so, just open the `.env` file under target and modify the variables. Restart the sample app, and you're ready to experiment using your own personalization content.
```

Comment:
Please take a look at the target READMEs and update them to reflect changes you made.  For example, I noticed the target personalization-hybrid README still says to "open the `.env` file at the root of this repository".  But that is no longer valid and should instead mention the `.env` file is within the `target` folder.